FMI-CROSS-CHECK-RULES.md

```diff
@@ -1,83 +1,56 @@
-# FMI Cross-Check: How to Improve FMI Compliance
+# FMI Cross-Check: How to Improve FMI Compliance, Version 4.1
 
 
-## 1. Motivation for Cross-Check Rules Changes for 4.0
+## 1. Motivation for Cross-Check Rules Changes to 4.1
 
-The growth of the Cross-Check (XC) repository and the result tables made it necessary to both rework the data handling concept and the display concepts.
-End of March 2018 a new infrastructure for the backend, data extraction and display was finalized reflecting roughly the Cross-Check Rules 3.1 with a number of improvements.
-What is new in XC 4.0:
+What is new in XC 4.1:
 
-- new repository structure and location
+- The FMI Steering Committee revised its decision to introduce fees for listing in the FMI Cross-Check tables.
+ 
+- The FMI Steering Committee decided to reduce the required proof for import results to the self-assessment passed/failed files.
+This will help reduce the space requirements for the Cross-Check repository without practically reducing the quality of the results.
+Existing result files for importers will be removed by the project factotum without further notice.
 
-- new rules what to store in the repository
 
-- new extraction process of the cross-check results
-
-- new display mechanisms for the cross-check results
-
-- new fee structure for showing tools at the cross-check site starting January 2019
-
-Fees are introduced to speedup the development of the FMI standard and further improve both the standard and tool implementations.
-We are planning to involve professional writers for the standard documents, pay for new reference FMUs and major FMI Compliance Checker improvements, contract XC infrastructure improvements and hire back-office support to free the predominantly technical team from menial office tasks.
-
-
-## 2. New Repository Structure
+## 2. Repository Structure
 
 To facilitate this process, the Modelica Association created the [FMI cross-check repository](https://github.com/modelica/fmi-cross-check) on GitHub.
-The data was clean up to facilitate automation.
-A continuous integration service is now in place to check any submission that violates the submission rules outlined here.
+A continuous integration service is in place to check any submission that violates the submission rules outlined here.
 
 
-## 3. New Rules what to store in the Repository
+## 3. Rules what to store in the Repository
 
 The growing number of FMUs provided makes it more and more important to automate cross-check activities for each vendor.
-The rules have now been tightened to allow automatic result comparison with specified epsilon bands.
+The rules have been tightened to allow automatic result comparison with specified epsilon bands.
```

Comment:
?? but we do not do the automatic comarison any more, right?

Reply:
Language fixed.